cni/log/logger.go

```diff
@@ -30,13 +32,18 @@ func initZapLog(logFile string) *zap.Logger {
 	encoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
 	jsonEncoder := zapcore.NewJSONEncoder(encoderConfig)
 
-	core := zapcore.NewCore(jsonEncoder, logFileCNIWriter, zapcore.DebugLevel)
-	Logger := zap.New(core)
-	return Logger
+	textFileCore := zapcore.NewCore(jsonEncoder, logFileCNIWriter, loggingLevel)
+	core, err := JoinPlatformCores(textFileCore, loggingLevel)
+	if err != nil {
+		// If we fail to join the platform cores, fallback to the original core.
+		core = textFileCore
+	}
+	Logger := zap.New(core, zap.AddCaller())
+	return Logger.With(zap.Int("pid", os.Getpid()))
```

Comment:
nit:
```suggestion
	return zap.New(core, zap.AddCaller()).With(zap.Int("pid", os.Getpid()))
```
no need for the extra local reference